app/views/survey_responses/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Index</h1>
+<table>
+  <tr>
+    <th>Survey Date</th>
+    <th>Confidence Level</th>
+    <th>Difficulty of Material</th>
+    <th>Notes</th>
+  </tr>
+
+<% @survey_responses.each do |survey_response| %>
+  <tr>
+    <td><%= survey_response.created_at.in_time_zone('Eastern Time (US & Canada)').strftime("%b %e, %Y") %></td>
```

Comment:
If you find yourself doing this in multiple places, it might be nice to refactor it into a method on either SurveyResponse or into a decorator. A decorator version might look something like this:

```
def created_at
  object.created_at.in_time_zone('Eastern Time (US & Canada)').strftime("%b %e, %Y")
end
```